skycoin/skywire
# Update/config ver

Did you run `make format && make check`?
yes

Fixes #	

 Changes:	
- Updated config version to `v1.1.1`

How to test this PR:
1. Start a visor with a `v1.1.0` config and see if the config is updated to `v1.1.1`
2. Generate a new config with `./skywire-cli config gen -r`
3. Run a visor with new config
